### DISCUSSION TITLE
Turn off anti-aliasing entirely

### DISCUSSION THREAD

**Discussion by HawkenKing** [google/model-viewer#4655] [Q&A]
Is it possible to completely remove anti-aliasing?

I checked the effect-composer msaa function, at 0 it retains the base antialiasing applied.

Is this something we can write custom JS to alter?

I'm using the pixelate-effect and the base anti-aliasing is noticeable and undesired "upscaled" into such an effect.

I tried adding `renderer = new THREE.WebGLRenderer( { antialias: false } );` to the script.js file, however this doesn't have any effect.

**Comment by elalish** [comment]
No, that's not part of our API. I would have expected your solution to work though - I think you're on the right track.
